lib/tasks/utils.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+// Include Gulp, plugins and local files
+var fs = require('fs');
+var handlebars = require('handlebars');
+var mkdirp = require('mkdirp');
+
+// Define Base folders 
+var srcDir = {
+    pages: '../../views/pages/',
+    posts: '../../views/posts/',
+    drafts: '../../views/drafts/',
+    layouts: '../../views/layouts/',
+    includes: '../../public/src/js/',
+    sass: '../../public/src/sass/*.scss',
+    dist: '../../dist/',
+    distCss: '../../dist/css/'
+};
+
+// siteData will hold the meta information for the site
+var siteData = {
+    posts: [],
+    pages: [],
+    archive: [],
+    tags: [],
+    navigation: []
+};
+
+// Functions
+var build = function () {
+    console.log('Build Initiated');
+};
+
+var createPage = function (data) {
```

Comment:
This is not a utility function and is very specific to the build process and should be in `build.js`